<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LpsXSim</string>
  </property>
  <property name="windowIcon">
   <iconset resource="lpsxsim.qrc">
    <normaloff>:/lpsxsim/icons/lpsxsim.ico</normaloff>:/lpsxsim/icons/lpsxsim.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <property name="margin">
     <number>3</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QGroupBox" name="transitionGroup">
        <property name="title">
         <string>Transitions</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="margin">
          <number>3</number>
         </property>
         <item>
          <widget class="QTableWidget" name="transitionTable">
           <property name="tabKeyNavigation">
            <bool>false</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>180</number>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string>Action/Probability</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>State Selector</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="stateGroup">
        <property name="title">
         <string>Current State</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="margin">
          <number>3</number>
         </property>
         <item>
          <widget class="QTableWidget" name="stateTable">
           <property name="tabKeyNavigation">
            <bool>false</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::NoSelection</enum>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string>Parameter</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Value</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QGroupBox" name="traceGroup">
       <property name="title">
        <string>Trace</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="margin">
         <number>3</number>
        </property>
        <item>
         <widget class="QTableWidget" name="traceTable">
          <property name="tabKeyNavigation">
           <bool>false</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>180</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>  #  </string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Action</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>State Change</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionLoadTrace"/>
    <addaction name="actionSaveTrace"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuPlay">
    <property name="title">
     <string>&amp;Play</string>
    </property>
    <addaction name="actionUndo_last"/>
    <addaction name="separator"/>
    <addaction name="actionPlayTrace"/>
    <addaction name="actionRandomPlay"/>
    <addaction name="actionStop"/>
    <addaction name="separator"/>
    <addaction name="actionSetPlayDelay"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>&amp;Options</string>
    </property>
    <addaction name="actionAutoSelectProbability"/>
    <addaction name="actionShowDontCaresInStateChanges"/>
    <addaction name="separator"/>
    <addaction name="actionOutput"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPlay"/>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="mcrl2::gui::qt::LogWidget" name="dockWidgetContents"/>
  </widget>
  <action name="actionPlayTrace">
   <property name="text">
    <string>Play &amp;Trace</string>
   </property>
  </action>
  <action name="actionRandomPlay">
   <property name="text">
    <string>&amp;Random Play</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>&amp;Stop</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionAutoSelectProbability">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Enable &amp;Autoselect Probability</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionLoadTrace">
   <property name="text">
    <string>&amp;Load Trace...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionSaveTrace">
   <property name="text">
    <string>&amp;Save Trace...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionShowDontCaresInStateChanges">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show &amp;Don't Cares in State Changes</string>
   </property>
  </action>
  <action name="actionSetPlayDelay">
   <property name="text">
    <string>Set Play &amp;Delay...</string>
   </property>
  </action>
  <action name="action_Fit_to_Current_State">
   <property name="text">
    <string>&amp;Fit to Current State</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionContents">
   <property name="text">
    <string>&amp;Contents</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionOutput">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show mCRL2 Output</string>
   </property>
  </action>
  <action name="actionUndo_last">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Undo last</string>
   </property>
   <property name="shortcut">
    <string>Backspace</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mcrl2::gui::qt::LogWidget</class>
   <extends>QWidget</extends>
   <header>mcrl2/gui/logwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="lpsxsim.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionUndo_last</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>undoLast()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>undoLast()</slot>
 </slots>
</ui>
